<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>播放/暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重复播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入歌曲名:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="title">
     <string>游戏模式</string>
    </property>
    <widget class="QPushButton" name="selectMusicFolderButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>选择 Music 文件夹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGameButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开始游戏 / 下一首</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="specificFolderRadioButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>指定文件夹</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="allFoldersRadioButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>所有文件夹</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>当前得分：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answerLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submitAnswerButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>591</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="duration1sRadioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>一秒</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="duration3sRadioButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>三秒</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="duration5sRadioButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>五秒</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="durationFullRadioButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>整首歌</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="hintListWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
